pkg/api/norman/customization/cluster/validator.go

```diff
@@ -384,8 +388,10 @@ func (v *Validator) validateEKSConfig(request *types.APIContext, cluster map[str
 	}
 
 	if !publicAccess && !privateAccess {
-		return httperror.NewAPIError(httperror.InvalidBodyContent,
-			"public access, private access, or both must be enabled")
+		if !importCreate {
+			return httperror.NewAPIError(httperror.InvalidBodyContent,
+				"public access, private access, or both must be enabled")
```

Comment:
could make the case that this should still fail

Reply:
Could you? When importing it is typical to only provide the few fields that are necessary to import the cluster since the spec will be overwritten anyway. This will partially change soon.